src/components/[guild]/crm/IdentitiesSearch.tsx

```diff
@@ -0,0 +1,76 @@
+import {
+  CloseButton,
+  Icon,
+  Input,
+  InputGroup,
+  InputLeftElement,
+  InputRightElement,
+} from "@chakra-ui/react"
+import { Column } from "@tanstack/react-table"
+import useDebouncedState from "hooks/useDebouncedState"
+import { MagnifyingGlass } from "phosphor-react"
+import { useEffect, useRef, useState } from "react"
+
+// export const identitiesFilter: FilterFn<any> = (row, columnId, value, addMeta) => {
+//   const { userId, addresses, platformUsers } = row.getValue(columnId) as Member
+
+//   if (
+//     userId === value ||
+//     platformUsers.some((platformAccount) =>
+//       platformAccount.username?.toLowerCase().includes(value)
+//     ) ||
+//     addresses.some((address) => address.toLowerCase().includes(value))
+//   )
+//     return true
+
+//   return false
+// }
+
+type Props = {
+  column: Column<any>
+}
+
+const IdentitiesSearch = ({ column }: Props) => {
+  const inputRef = useRef<HTMLInputElement>()
+  const [localValue, setLocalValue] = useState(column.getFilterValue() as string)
```

Comment:
I tried to find a solution to type these properly, but seems like the `getFilterValue` function always returns `unknown`, right? In that case, I can't really find a better solution than this.